testsrc/test262.properties

```diff
@@ -6151,8 +6145,8 @@ language/statements/generators 220/259 (84.94%)
     param-dflt-yield.js {unsupported: [default-parameters]}
     params-dflt-args-unmapped.js {unsupported: [default-parameters]}
     params-dflt-ref-arguments.js {unsupported: [default-parameters]}
-    params-trailing-comma-multiple.js
-    params-trailing-comma-single.js
+    params-trailing-comma-multiple.js non-interpreted
```

Comment:
Does this mean it fails in compiled mode?

Reply:
The testcase fails due to #1412 now, but the params-trailing-comma part is fixed